htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -892,6 +892,7 @@ public function fetchAllByAccount($sortorder = '', $sortfield = '', $limit = 0,
 		// Affichage par compte comptable
 		if (!empty($option)) {
 			$sql .= ' AND t.subledger_account IS NOT NULL';
+			$sql .= ' AND t.subledger_account != ""';
```

Comment:
String for sql must always be '    when you can use both   ' or  " for php string
So use instead
`$sql .= " AND t.subledger_account != ''";`